overthebox/files/usr/share/otb/post-tracking.d/get-ip-asn

```diff
@@ -15,7 +15,13 @@ _curl() {
 }
 
 pub_ip=$(_curl ifconfig.ovh)
```

Comment:
True, but what if curl succedes and return invalid data ?

Small -z and != null checks are easy to do =)